Bottom: 4bd219d3eae193abfef2eacc8d4a573e7d6b7fd6
Top:    ad20f06180db60d742b3b5caea7c40271d22d810
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2011-12-26 16:26:52 +0000

djopt.c: Fix incorrect allocation.

I missed this fix from the commit ff2492399fd4743c3fd89ffd4fe6650ad0d80a4a

(Caught by coverity)

Coverity-cid: 239


---

diff --git a/src/djopt.c b/src/djopt.c
index 8a51d0e..eae52c6 100644
--- a/src/djopt.c
+++ b/src/djopt.c
@@ -1297,7 +1297,7 @@ orthopull_1 (corner_s * c, int fdir, int rdir, int any_sel)
       if (cn >= cm)
 	{
 	  cm = cn + 10;
-	  cs = (corner_s **) realloc (cs, cm * sizeof (corner_s));
+	  cs = (corner_s **) realloc (cs, cm * sizeof (corner_s *));
 	}
       cs[cn++] = c2;
       r2 = corner_radius (c2);
